livingreviews/.htaccess

```
@@ -2,4 +2,54 @@ Header set Access-Control-Allow-Origin *
 Options +FollowSymLinks
 RewriteEngine on
 
-RewriteRule ^$ https://networkinstitute.org/projects/mapping-communication-science-with-living-literature-reviews/ [R=302,L]
+RewriteRule ^$ https://github.com/timolek/living_reviews/ [R=302,L]
+
+# Definitions
+
+# Latest pointing to the most recent ones
+RewriteRule ^np/vocab/latest/livingLiteratureVocab$ http://purl.org/np/RAZ0qV54EeMo-P1QvcYRSf6OSzu6rVEuRbprs3xM7gpKE [R=302,L]
+RewriteRule ^np/vocab/latest/EmpiricalArticle$ http://purl.org/np/RApVFe9whuJp4U8ZBGW7MlmGxXumyPNx-S1Xpat1rrwiI [R=302,L]
+RewriteRule ^np/vocab/latest/ConceptPaper$ http://purl.org/np/RANPaYkBj85yyphoroDrtygS5XgIT1VBnwXI0c6g0M_EU [R=302,L]
+RewriteRule ^np/vocab/latest/LiteratureReview$ http://purl.org/np/RAgrLSXcIap-YK8OMdTAWpSZnohD508BmDxDfjBVKjvMs [R=302,L]
+RewriteRule ^np/vocab/latest/QuantativeAnalysis$ http://purl.org/np/RAGpM3_6b7Bx-XdargatFgS72NEAAU1LWxYuk-F_bhB4g [R=302,L]
+RewriteRule ^np/vocab/latest/Mixed-Method-Design$ http://purl.org/np/RATlMS4SYWoEVCfL_c6B8hx2wCdiEyBbP_OJJfrrAAvFc [R=302,L]
+RewriteRule ^np/vocab/latest/Survey$ http://purl.org/np/RAW4AzXLO_JqAIHXB74-SB-vw9emY5_qZGYqUB845H4hI [R=302,L]
+RewriteRule ^np/vocab/latest/Experiment$ http://purl.org/np/RAofY9W3DMmfH9ZrR0FQXjLpvY9p7f6j3h1Xn4foyMWqc [R=302,L]
+RewriteRule ^np/vocab/latest/ContentAnalysis$ http://purl.org/np/RAr_syyRjT-m2gIikaftjMT359bqZYzNwrg5qMX_4oIRk [R=302,L]
+RewriteRule ^np/vocab/latest/Observation$ http://purl.org/np/RA2wF9aXK7__iVASkS3zkl0BhvO5SbAqwySS_lof2CwfY [R=302,L]
+RewriteRule ^np/vocab/latest/providesEvidenceFor$ http://purl.org/np/RAHchM56lzBUP4TNmoilnFOkww_fqwYbQbXtjKtbisC2Q [R=302,L]
+RewriteRule ^np/vocab/latest/providesCounterEvidenceFor$ http://purl.org/np/RAr6odCvoXemRYXloInpsOAlNGU2WvtMnmjMNjxOTmEGM [R=302,L]
+RewriteRule ^np/vocab/latest/landOfFocus$ http://purl.org/np/RAuwKmZrKQo5Eyl1Ng_QQBDOlJDFJeRjWa4bNc2LpUkPM [R=302,L]
+RewriteRule ^np/vocab/latest/socialMediaPlatform$ http://purl.org/np/RAvBxA3L1yVULzfGuqAJQPkpw0qqs-fKSiJKuGh4g6X8Y [R=302,L]
+RewriteRule ^np/vocab/latest/theoreticalApproach$ http://purl.org/np/RARF4ONG0BW7G2OsaIptytnhjP_LSN6213SCoONcgcapc [R=302,L]
+RewriteRule ^np/vocab/latest/primaryObject$ http://purl.org/np/RARqaQFaX_ME24YfMPi4yQ_PgCJRXhE6oZnAumIz1hHDs [R=302,L
+RewriteRule ^np/vocab/latest/firstAuthorOrigin$ http://purl.org/np/RA72Jm8VDYgT2CZ2snyrxKJ82T4flD1In8mZdqH_ooxnQ [R=302,L]
+
+# Used namespace
+RewriteRule ^vocab//livingLiteratureVocab$ https://w3id.org/livingreviews/np/vocab/latest/livingLiteratureVocab [R=302,L]
```

I'm pretty sure the error you spotted is the double solidus ... but if not, it is helpful to state both what needs to be changed and what it needs to change to, if not to use the github "suggest" feature, as I did here.
```suggestion
RewriteRule ^vocab/livingLiteratureVocab$ https://w3id.org/livingreviews/np/vocab/latest/livingLiteratureVocab [R=302,L]
```